Packs/UnifiVideoNVR/Integrations/UnifiVideo/UnifiVideo.py

```diff
@@ -13,6 +13,32 @@
 port = params.get('port')
 schema = params.get('schema')
 verify_cert = params.get('verify_cert')
+FETCH_TIME = params.get('fetch_time')
+TIME_UNIT_TO_MINUTES = {'minute': 1, 'hour': 60, 'day': 24 * 60, 'week': 7 * 24 * 60, 'month': 30 * 24 * 60,
+                        'year': 365 * 24 * 60}
+
+
+def parse_time_to_minutes():
```

Comment:
you can use `parse_date_range` function from commonServerPython instead.
We have a lot of integrations that uses this function, for example, `McAfee_ESM_v2`

Reply:
I've copy-pasted this from spunkPy.